packages/eds-core-react/src/components/Menu/MenuItem.tsx

```diff
@@ -101,6 +101,8 @@ export type MenuItemProps = {
   disabled?: boolean
   /** onClick handler */
   onClick?: (e: React.MouseEvent) => void
+  /** onClose handler */
+  onClose?: (e: React.KeyboardEvent) => void
```

Comment:
Suggest removing this as we already have an `onClose` passed on top level and it deviates from intended expectation with click vs. keydown.